lib/store/actions.js

```diff
@@ -242,6 +244,42 @@ const actions = {
       },
     };
   },
+
+  startWatchingLocation() {
+    return (dispatch) => {
+      dispatch(actions.locationWatchingStarted());
+      watchID = navigator.geolocation.watchPosition(
```

Comment:
This should never happen because start and stop watching are called during the component lifecycles.
The geolocation is not watched anywhere else in the application.